bin/cylc

```diff
@@ -31,7 +31,7 @@ print_version() {
     if [[ "$#" -eq 0 ]]; then
         echo "$CYLC_VERSION"
     fi
-    if [[ "$@" == 'long' || "$@" == '--long' ]]; then
+    if [[ "$*" == 'long' || "$*" == '--long' ]]; then
```

Comment:
I wonder why the linter has a problem with `"$@"`?

Reply:
So `$@` is an array whereas `$*` is a string so this syntactically a bit odd.

https://github.com/koalaman/shellcheck/wiki/SC2124